README.md

```diff
@@ -1,6 +1,43 @@
-# immutad●t
+immutad●t
+=================
+
+A JavaScript library to deal with nested immutable structures.
+
+```js
+push({ nested: { prop: [1, 2] } }, 'nested.prop', 3, 4)
+// → { nested: { prop: [1, 2, 3, 4] } }
+
+pickBy({ nested: [{ a: 1, b: 2, c: 3, d: 4 }, { e: 6 }] }, 'nested.0', v => v < 3)
+// → { nested: [{ a: 1, b: 2 }, { e: 6 }] }
+```
+immutad●t lets you use a more meaningful and shorter syntax to apply operations on immutable structures.
```

Comment:
Maybe put "short" before "meaningful", because "Concise" is the first aspect in "Our approch"